datalad_debian/new_distribution.py

```diff
@@ -27,6 +27,12 @@
 @build_doc
 class NewDistribution(Interface):
     """Create a new distribution dataset
+
+    A typical distribution dataset contains a 'builder' subdataset with one or
+    more build environments, and a package subdirectory with one subdataset per
```

Comment:
Comment rather than a suggestion: This comma technically should not be there, as these are not independent clauses. But it reads easier (for me) with the comma present.